<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FolderMoveCopy</class>
 <widget class="QMainWindow" name="FolderMoveCopy">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>794</width>
    <height>596</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FolderMoveCopy</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLineEdit" name="originEdit">
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="placeholderText">
         <string>源文件（夹）</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="originBtn">
        <property name="text">
         <string>选择源文件（夹）</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLineEdit" name="targetEdit">
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="placeholderText">
         <string>目标文件夹</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="targetBtn">
        <property name="text">
         <string>选择目标文件夹</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QWidget" name="radioWidget" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="leftMargin">
        <number>6</number>
       </property>
       <property name="topMargin">
        <number>6</number>
       </property>
       <property name="rightMargin">
        <number>6</number>
       </property>
       <property name="bottomMargin">
        <number>6</number>
       </property>
       <item>
        <widget class="QRadioButton" name="copyRbn">
         <property name="text">
          <string>复制</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="moveRbn">
         <property name="text">
          <string>移动</string>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="folderLabel">
         <property name="text">
          <string>若文件夹已存在：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="folderCbB"/>
       </item>
       <item>
        <widget class="Line" name="line_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="fileLabel">
         <property name="text">
          <string>若文件已存在：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="fileCbB"/>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>146</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="moveCopyBtn">
         <property name="text">
          <string>复制</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="CodeEditor" name="plainTextEdit"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="clearBtn">
        <property name="text">
         <string>清空</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>794</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="fileMenu">
    <property name="title">
     <string>&amp;文件</string>
    </property>
    <addaction name="originAct"/>
    <addaction name="fromTxtAct"/>
    <addaction name="fromClipboardAct"/>
    <addaction name="separator"/>
    <addaction name="targetAct"/>
    <addaction name="separator"/>
    <addaction name="moveCopyAct"/>
    <addaction name="separator"/>
    <addaction name="clearAct"/>
    <addaction name="separator"/>
    <addaction name="quitAct"/>
   </widget>
   <widget class="QMenu" name="editMenu">
    <property name="title">
     <string>&amp;编辑</string>
    </property>
   </widget>
   <widget class="QMenu" name="helpMenu">
    <property name="title">
     <string>&amp;帮助</string>
    </property>
    <addaction name="aboutAct"/>
    <addaction name="aboutQtAct"/>
   </widget>
   <addaction name="fileMenu"/>
   <addaction name="editMenu"/>
   <addaction name="helpMenu"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>工具栏</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="originAct"/>
   <addaction name="fromTxtAct"/>
   <addaction name="fromClipboardAct"/>
   <addaction name="separator"/>
   <addaction name="targetAct"/>
   <addaction name="separator"/>
   <addaction name="clearAct"/>
   <addaction name="separator"/>
   <addaction name="moveCopyAct"/>
  </widget>
  <action name="originAct">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/document-folder.svg</normaloff>:/icons/document-folder.svg</iconset>
   </property>
   <property name="text">
    <string>选择&amp;源文件（夹）</string>
   </property>
   <property name="toolTip">
    <string>选择源文件（夹）</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="targetAct">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/document-open.svg</normaloff>:/icons/document-open.svg</iconset>
   </property>
   <property name="text">
    <string>选择&amp;目标文件夹</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
  </action>
  <action name="fromTxtAct">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/document-new.svg</normaloff>:/icons/document-new.svg</iconset>
   </property>
   <property name="text">
    <string>从&amp;文本文件导入</string>
   </property>
  </action>
  <action name="fromClipboardAct">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/edit-paste.svg</normaloff>:/icons/edit-paste.svg</iconset>
   </property>
   <property name="text">
    <string>从&amp;剪贴板导入</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+V</string>
   </property>
  </action>
  <action name="moveCopyAct">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/edit-copy.svg</normaloff>:/icons/edit-copy.svg</iconset>
   </property>
   <property name="text">
    <string>复制</string>
   </property>
   <property name="toolTip">
    <string>复制/移动</string>
   </property>
  </action>
  <action name="quitAct">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/quit.svg</normaloff>:/icons/quit.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;退出</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="aboutAct">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/help-browser.svg</normaloff>:/icons/help-browser.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;关于</string>
   </property>
  </action>
  <action name="aboutQtAct">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/qtlogo.svg</normaloff>:/icons/qtlogo.svg</iconset>
   </property>
   <property name="text">
    <string>关于 &amp;Qt</string>
   </property>
  </action>
  <action name="copyAct">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/edit-copy.svg</normaloff>:/icons/edit-copy.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;复制</string>
   </property>
  </action>
  <action name="moveAct">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/move.svg</normaloff>:/icons/move.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;移动</string>
   </property>
   <property name="toolTip">
    <string>移动</string>
   </property>
  </action>
  <action name="clearAct">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/clear.svg</normaloff>:/icons/clear.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;清空</string>
   </property>
   <property name="shortcut">
    <string>Alt+C</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>CodeEditor</class>
   <extends>QPlainTextEdit</extends>
   <header location="global">codeeditor.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
